src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -208,15 +218,21 @@ def _try_handle_high_volume_task_trigger(ts: datetime):
     _dispatch_tasks(ts)
 
 
-def _process_message(ts: datetime, wrapper: CheckinMessage) -> None:
-    # When running in high volume mode we will not consume clock pulses (The
-    # clock_pulse task is not enabled). Instead we use each check-in message as
-    # a means for triggering our tasks.
-    if settings.SENTRY_MONITORS_HIGH_VOLUME_MODE:
-        try:
-            _try_handle_high_volume_task_trigger(ts)
-        except Exception:
-            logger.exception("Failed try high-volume task trigger", exc_info=True)
+def _process_message(ts: datetime, wrapper: CheckinMessage | ClockPulseMessage) -> None:
+    # XXX: Relay does not attach a message type, to properly discriminate the
+    # type we add it by default here. This can be removed once the message_type
+    # is guaranteed
+    if "message_type" not in wrapper:
+        wrapper["message_type"] = "check_in"
+
+    try:
+        _try_monitor_tasks_trigger(ts)
+    except Exception:
+        logger.exception("Failed to trigger monitor tasks", exc_info=True)
+
+    # Nothing else to do with clock pulses
+    if wrapper["message_type"] == "clock_pulse":
+        return
 
     params: CheckinPayload = json.loads(wrapper["payload"])
     start_time = to_datetime(float(wrapper["start_time"]))
```

Comment:
do we need to cast if we update the typing?

Reply:
Actually I suppose I'm not 100% sure on this, so would rather leave it